.storybook/preview.js

```diff
@@ -41,12 +41,12 @@ export const parameters = {
     },
   },
   exportToSandbox: {
-    provider: 'codesandbox-browser',
-    bundler: 'cra',
+    provider: 'stackblitz-cloud',
+    bundler: 'vite',
     requiredDependencies: {
       // for React
-      react: '^17',
-      'react-dom': '^17',
+      react: '^18',
```

Comment:
wondering if  this is a good/stable idea ? we dont have proper coverage against react 18. 

while this might introduce a "more bug reports from users" opportunity, I dont think it's the best way to "test" our code against  v18.

Reply:
Release of React 19 is coming, Teams and other partners AFAIK are pushing React 18 upgrades. Also, more problems are happening with `17 => 18`, but I am not aware of any when code for 18 breaks for 17.

P.S. This might be a good topic for v-build, but IMO once only v9 will be on `master`, we should switch to React 18.